### ISSUE TITLE
what am I doing wrong?

### ISSUE THREAD

**Issue by zmiler2** [canonical/lxd-ui#773]
```
Error: Could not find physical in networkTypeToOptionKey
    at eG (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:87:30823)
    at ie (https://18.34.0.18:50443/ui/assets/FormMenuItem-f4d64986.js:1:5988)
    at te (https://18.34.0.18:50443/ui/assets/FormMenuItem-f4d64986.js:1:4865)
    at be (https://18.34.0.18:50443/ui/assets/FormMenuItem-f4d64986.js:1:7865)
    at J (https://18.34.0.18:50443/ui/assets/NetworkForm-34d3abf4.js:1:4879)
    at Zg (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:38:19529)
    at B1 (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:40:44027)
    at L1 (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:40:39763)
    at m$ (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:40:39691)
    at Cd (https://18.34.0.18:50443/ui/assets/index-b3f3eaed.js:40:39545)
```

**Comment by edlerd** [comment]
Thank you for reaching out! You are doing everything right. This is a known bug, and it has been reported and fixed very recently in https://github.com/canonical/lxd-ui/issues/757

We plan to release a new Version of the UI with a fix into 5.21 LTS shortly.

**Comment by edlerd** [comment]
closing as a duplicate of  #787
